lib/selftest/src/Obelisk/SelfTest.hs

```diff
@@ -133,25 +146,25 @@ main = do
             revParseHead
 
           thunk  = ".obelisk/impl"
-          update = run "ob" ["thunk", "update", thunk] >> commitAll
-          pack   = run "ob" ["thunk", "pack",   thunk] >> commitAll
-          unpack = run "ob" ["thunk", "unpack", thunk] >> commitAll
+          update = runOb ["thunk", "update", thunk] >> commitAll
+          pack   = runOb ["thunk", "pack",   thunk] >> commitAll
+          unpack = runOb ["thunk", "unpack", thunk] >> commitAll
 
           diff a b = run "git" ["diff", a, b]
 
       describe "ob init" $ parallel $ do
-        it "works with default impl"       $ inTmp $ \_ -> run "ob" ["init"]
-        it "works with master branch impl" $ inTmp $ \_ -> run "ob" ["init", "--branch", "master"]
-        it "works with symlink"            $ inTmp $ \_ -> run "ob" ["init", "--symlink", toTextIgnore obeliskImpl]
+        it "works with default impl"       $ inTmp $ \_ -> runOb ["init"]
+        it "works with master branch impl" $ inTmp $ \_ -> runOb ["init", "--branch", "master"]
+        it "works with symlink"            $ inTmp $ \_ -> runOb ["init", "--symlink", toTextIgnore obeliskImpl]
         it "doesn't silently overwrite existing files" $ withSystemTempDirectory "ob-init λ" $ \dir -> do
-          let p force = (System.Process.proc "ob" $ "--no-handoff" : "init" : ["--force"|force]) { cwd = Just dir }
+          let p force = (System.Process.proc ob $ "--no-handoff" : "-v" : "init" : ["--force"|force]) { cwd = Just dir }
```

Comment:
Since this one was just used once and was hidden by default I just hardcoded it. Let me verify the signature of `System.Process.proc` to see if it is the same and can be wrapped easily.

Reply:
Looks like System.Process.proc needs a `[String]` for the args...

I think we can make `convertRunner` support both by adding a type variable with an IsString constraint:

```
convertRunner :: IsString a => (String -> [a] -> b) -> String -> Bool -> [a] -> b
convertRunner runner command isVerbose args = runner command $ (if isVerbose then ("-v" :) else id) args
```

Reply:
But if we implement the latter part of [this comment](https://github.com/obsidiansystems/obelisk/pull/609#discussion_r363898842), we'd be able to remove the IsString constraint